### DISCUSSION TITLE
Unable to move file from GCS to BigQuery

### DISCUSSION THREAD

**Discussion by rsotogar** [apache/airflow#12279] [Q&A] [answered]
I am getting this error message when executing my dag. It is a very simple task to get initiated with Airflow workflows:

```Exception: BigQuery job failed. Final error was: {'reason': 'notFound', 'message': 'Not found: URI gs://modulo_spark_bucket/l'}```

And this is my dag:

```
default_args = {
    'owner': 'airflow',
    'depends_on_past': False,
    'start_date': days_ago(1),
    'email': ['sotogarcia.r@icloud.com'],
    'email_on_failure': True,
    'email_on_retry': False,
    'retries': 0,
    'retry_delay': timedelta(minutes = 5),
    # 'queue': 'bash_queue',
    # 'pool': 'backfill',
    # 'priority_weight': 10,
    # 'end_date': datetime(2016, 1, 1),
    # 'wait_for_downstream': False,
    # 'dag': dag,
    # 'sla': timedelta(hours=2),
    # 'execution_timeout': timedelta(seconds=300),
    # 'on_failure_callback': some_function,
    # 'on_success_callback': some_other_function,
    # 'on_retry_callback': another_function,
    # 'sla_miss_callback': yet_another_function,
    # 'trigger_rule': 'all_success'
}

#define task

with DAG(
    'GCS_toBQtable',
    default_args=default_args,
    description='Data transfer to a new BQ table',
    schedule_interval=None,
) as my_dag:
    

    t1 = GoogleCloudStorageToBigQueryOperator(task_id = "GCS_to_BQ",
        bucket = "modulo_spark_bucket", 
        source_objects = "gs://modulo_spark_bucket/data/pokemon.csv", 
        autodetect = True,
        skip_leading_rows = 1,
        create_disposition = "CREATE_IF_NEEDED",
        destination_project_dataset_table = "neural-theory-277009.pokemon_data.pokemons")
    


#dependencies

t1

```

As you see the error message indicates a problem when accessing my storage bucket, in particular a source file: "gs://modulo_spark_bucket/l'" that does not exist. Is there a problem with my dag? I am creating the script in my local Conda environment and then uploading it as a .py file to cloud composer. Airflow version is 1.10.12+composer.

**Comment by turbaszek** [comment, marked as answer]
@rsotogar try:
```
bucket='modulo_spark_bucket',
source_objects=['modulo_spark_bucket/data/pokemon.csv'],
```

Here's latest docs https://airflow.readthedocs.io/en/latest/howto/operator/google/cloud/gcs.html#gcstobigqueryoperator for `GCSToBigQueryOperator` which is available in providers package.

**Reply by leahecole** [reply]
Marking as answer because I'm reasonably confident this is the answer, but if it's not, feel free to reopen (if that's a thing in Discussions?) or re-ask 😄

**Comment by turbaszek** [comment]
@rsotogar have you been able to do the upload? 😉
